### DISCUSSION TITLE
What can I do with this redirection?

### DISCUSSION THREAD

**Discussion by MatiDiez** [cypress-io/cypress#15452] [Questions and Help]
Hey guys. I'm no expert on this but I've read that frame busters kill cypress tests in some scenarios. 
I'm facing a proble while running a test that has a redirection. The test immediately throws in "Whoops there is no test to run" after I click on a button. 

This button has many functions but a piece of the code says: **window.top.location = url || window.CALLBACK_URL** which kinda looks like this problem. 

Is there a work around for this besides simply disabling it?

**Comment by bahmutov** [comment]
It does look like a problem, and usually we detect this via https://docs.cypress.io/guides/references/configuration.html#modifyObstructiveCode but in your case something might not work as expected. Creating a reproducible example would be a good way to see what is going on

**Reply by MatiDiez** [reply]
Yeah it is hard to come up with a running example. But I could find more info. Specifically the test breaks when doing this: 

GET /emails/16320896044107446/editor 302 34.000 ms - 52
GET /__/ 200 0.768 ms - -

Because it is a different application that has this: window.top.location = url || window.CALLBACK_URL

I'm trying to come up with a workaround but I've been at this for two days already >.<
